src/server/scripts/EasternKingdoms/Karazhan/bosses_opera.cpp

```diff
@@ -793,9 +793,11 @@ enum RedRidingHood
     SPELL_WIDE_SWIPE                = 30761,
 
     CREATURE_BIG_BAD_WOLF           = 17521,
-};
 
-#define GOSSIP_GRANDMA          "What phat lewtz you have grandmother?"
+    // Texts
+    GOSSIP_MENU_GRANDMOTHER         = 7441,
+    GOSSIP_OPTION_GRANDMOTHER       = 0
```

Comment:
just use `GOSSIP_ICON_CHAT` for this

Reply:
`GOSSIP_ICON_CHAT` is not really a descriptive name for this and could confuse people reading the code.
A workaround to get rid of `GOSSIP_OPTION_GRANDMOTHER` would be to introduce a new constant named something like `GOSSIP_OPTION_DEFAULT`, but this would make the code less maintainable.